webrtc.html

```diff
@@ -8852,6 +8854,34 @@ <h2 id="sec.stats-model">Statistics Model</h2>
       [[!WEBRTC-STATS]].</p>
     </section>
     <section>
+      <h3>The stats selection algorithm</h3>
+      <p>
+	The <dfn>stats selection algorithm</dfn> is as follows:
+	<ul>
+	  <li>If the selector is a MediaStreamTrack, return the following objects:
```

Comment:
I scratched my head about that. It seems that pulling in everything it references or that has references to (RTP, track) has two issues:
1) It gets a lot (pull in streams, you get references to the other tracks in the stream; should you pull those? pull in what RTP references, you get the transport, which gets the candidates..)
2) It isn't extension-friendly (add a new reference; do you pull what it points to or not?)